tests/test_dockhub.py

```diff
@@ -0,0 +1,44 @@
+from click.testing import CliRunner
+
+from dockhub import main, DOCKERHUB_URL
+
+
+def test_basic():
+    """Verify that it runs and spits out help."""
+    runner = CliRunner()
+    result = runner.invoke(main, "--help")
+    assert result.exit_code == 0
```

Comment:
Doesn't this test look beautiful? click and pytest make writing tests straight-forward without a lot of effort around getting the universe into alignment to set up the test conditions.